libs/ui/slider/brain/package.json

```diff
@@ -2,12 +2,12 @@
 	"name": "@spartan-ng/ui-slider-brain",
 	"version": "0.0.1-alpha.357",
 	"peerDependencies": {
-		"@spartan-ng/ui-label-brain": "0.0.1-alpha.357",
 		"@angular/common": ">=18.0.0",
 		"@angular/core": ">=18.0.0",
 		"@angular/cdk": ">=18.0.0",
 		"@angular/forms": ">=18.0.0",
-		"rxjs": ">=6.6.0"
+		"rxjs": ">=6.6.0",
+		"@spartan-ng/brain": "0.0.1-alpha.357"
```

Comment:
is there a way to order these automatically as part of the migration?

Reply:
I think this was added by ESLint Nx dependency checks rule rather than the migration itself in this case.

That being said I could add this prettier plugin https://github.com/matzkoh/prettier-plugin-packagejson to always ensure our deps are sorted correctly if you'd like?

Reply:
Up to you. I mean it's a nice to have, but doesn't really matter too much. I'll trust your judgement!

Reply:
Pushed a commit there now including it, we can give it a go, if its not what we want we can always remove it I suppose.